### PR TITLE
Support font features

### PR DESCRIPTION
**Pull request by anood**

### CODE REVIEW

File: scribus/ui/propertywidget_opentypefontfeatures.cpp [hunk 1]
```diff
@@ -43,16 +43,70 @@ void PropertyWidget_OpenTypeFontFeatures::showFontFeatures(QString s)
 {
 	if (!m_ScMW || m_ScMW->scriptIsRunning())
 		return;
-	fontfeatures->setText(s);
```
Comment (khaledhosny) [inline scribus/ui/propertywidget_opentypefontfeatures.cpp:46]: Why was this code removed, this method does not do anything now.

File: scribus/ui/propertywidget_opentypefontfeatures.cpp [hunk 2]
```diff
@@ -110,17 +164,66 @@ void PropertyWidget_OpenTypeFontFeatures::updateStyle(const ParagraphStyle& newC
...
+	disconnect(DefaultFractionsRadio, SIGNAL(clicked()), this, SLOT(handlefontfeatures()));
+	disconnect(DiagonalRadio, SIGNAL(clicked()), this, SLOT(handlefontfeatures()));
+	disconnect(StackedRadio, SIGNAL(clicked()), this, SLOT(handlefontfeatures()));
+	disconnect(OrdinalCheck, SIGNAL(clicked()), this, SLOT(handlefontfeatures()));
+	disconnect(SlashedZeroCheck, SIGNAL(clicked()), this, SLOT(handlefontfeatures()));
 }
 
 void PropertyWidget_OpenTypeFontFeatures::languageChange()
 {
-	fontfeatures->setText(tr("FontFeatures"));
```
Comment (khaledhosny) [inline scribus/ui/propertywidget_opentypefontfeatures.cpp:123]: Why was this line removed leaving an empty method?

File: scribus/ui/propertywidget_opentypefontfeatures.cpp (hunk 1 above)
Comment (asmaAL-Bahanta) [inline scribus/ui/propertywidget_opentypefontfeatures.cpp:46]: I found that this function is called from `ScribusMainWindow::setItemFontFeatures` which is not used or called anywhere. Is it ok if i removed the whole function ?!

Comment (khaledhosny) [inline scribus/ui/propertywidget_opentypefontfeatures.cpp:46]: If the code works fine (font features are saved/read from file and get applied to the current text frame as expected) then just remove the unused code, otherwise find where it should be used and use it.

**Comment by khaledhosny** [comment]
Obsoleted by #132.
